src/stylesheets/components/_hide-this-page.scss

```diff
@@ -0,0 +1,86 @@
+@media only screen {
+    .js-enabled .govuk-quick-exit__container.govuk-visually-hidden {
```

Comment:
Apologies if I'm not reading this correctly - does this mean that this quick exit component will only work for users that have JavaScript enabled?

Reply:
If JS is not enabled, the user **won't** get the functionality that opens BCC Weather in a new tab, however, it will still redirect them to Google. This would still be considered a safe action, compared to if it did nothing 😄

Reply:
Ah, I understand - this is to hide container that has the text "To leave the page quickly, press the escape key" as the escape key won't work without JavaScript. Super.